MAINTAINERS.md

```diff
@@ -19,10 +19,10 @@ Before the review period starts, the previous version should be re-released with
 
 # Pull Requests
 
-You should use [homu](http://homu.io) when accepting pull requests.
+You should use [homu](https://github.com/barosl/homu) when accepting pull requests.
 You can read about the available commands in the front page.
 
-The Guides repository has homu [configured to auto-squash commits](http://homu.io/r/emberjs/guides).
+The Guides repository has homu configured to auto-squash commits.
```

Comment:
We don't actually use any bot at the moment, though we could try setting up bors-ng.